Dashboard/app/js/templates/navReports/export-reports.handlebars

```diff
@@ -30,50 +31,56 @@
         document.getElementById(exportName).style.display = "block";
         evt.currentTarget.className += " active";
        }
-    </script> 
+    </script>      
     <section class="exportContainer">
       <ul class="exportSelect">
         <li class="dataCleanExp trigger" onclick="openExportOptions(event, 'dataCleanExp_options')">
           <h2>{{t _data_cleaning_export}}</h2>
           <h6>{{t _importable_back_to_akvo_flow}}</h6>
           <p class="expDescr">{{t _combines_options}}</p>
+        {{#view FLOW.ExportReportsAppletView dataCleaningSection=true}}
           <div id="dataCleanExp_options" class="options">
               <div class="dataCollectedDate">
-                <p>{{t _collection_period}}</p>
+                <p>{{t _collection_period}}</p>  
```

Comment:
remove whitespace

Reply:
@muloem most of the issues were actually not present for me locally but made a commit for those that were present

Reply:
there was no white space here either